<ui version="4.0" >
 <comment>
*********************************************************************************
**
**   $Id$
**   Copyright (c) 2004 Joe Croft joe@croftj.net
**   
**   This file is part of ah2.
**
**   Ah3 is free software; you can redistribute it and/or modify
**   it under the terms of the GNU General Public License as published by
**   the Free Software Foundation; either version 2 of the License, or
**   (at your option) any later version.
**
**   Ah3 is distributed in the hope that it will be useful,
**   but WITHOUT ANY WARRANTY; without even the implied warranty of
**   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
**   GNU General Public License for more details.
**
**   You should have received a copy of the GNU General Public License
**   along with Ah3; if not, write to the Free Software
**   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
**
*********************************************************************************
</comment>
 <class>VendorSelect</class>
 <widget class="QDialog" name="VendorSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>237</red>
        <green>237</green>
        <blue>237</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>147</red>
        <green>147</green>
        <blue>147</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>110</red>
        <green>110</green>
        <blue>110</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Vendor Select</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QcjDataTable" name="vendorTable" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string>&lt;html>This table is for selecting a vendor. It's xml form name is &lt;em>vendor_select_table&lt;/em>&lt;/html></string>
     </property>
     <property name="xml_definition" stdset="0" >
      <string>vendor_select_table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyBtn" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcjDataTable</class>
   <extends>QFrame</extends>
   <header>QcjData/QcjDataTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
